src/Goerli-DssSpell.sol

```diff
@@ -106,6 +102,7 @@ contract DssSpellAction is DssAction {
             "RWA007-A",
              500 * MILLION * WAD
         );
+        Pricing(DssExecLib.spotter()).poke("RWA007-A");
```

Comment:
this can be done using `DssExecLib`, no need to add interfaces and extra steps

Reply:
I want to safe some gas, as we already increase dc using autoline

Reply:
done